osu.Game.Tests/Editing/Checks/CheckAbnormalDifficultySettingsTest.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Edit.Checks;
+using NUnit.Framework;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Tests.Beatmaps;
+using System.Linq;
+
+namespace osu.Game.Tests.Editing.Checks
+{
+    [TestFixture]
+    public class CheckAbnormalDifficultySettingsTest
+    {
+        private CheckAbnormalDifficultySettings check = null!;
+
+        private IBeatmap beatmap = new Beatmap<HitObject>();
+
+        [SetUp]
+        public void Setup()
+        {
+            check = new CheckAbnormalDifficultySettings();
+            beatmap.Difficulty = new();
```

Comment:
Please don't use the `new()` syntax, we don't use it here. Full type names when calling ctor please.

Reply:
Understood, i will be closing this conversation since this test is supposed to be deleted and replaced by ruleset specific ones.